src/promptflow-devkit/promptflow/_sdk/entities/_flows/base.py

```diff
@@ -145,6 +146,7 @@ def __init__(
         **kwargs,
     ):
         self.variant = kwargs.pop("variant", None) or {}
+        self._log_level = kwargs.pop("log_level", None)
```

Comment:
I think this setting should not be in Flow obj as it's changes static loggers which is not part of flow